{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2019-5636",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "https:\/\/blog.rapid7.com\/2019\/10\/08\/r7-2019-32-denial-of-service-vulnerabilities-in-beckhoff-twincat-plc-environment-fixed\/",
                    "name": "https:\/\/blog.rapid7.com\/2019\/10\/08\/r7-2019-32-denial-of-service-vulnerabilities-in-beckhoff-twincat-plc-environment-fixed\/",
                    "refsource": "MISC",
                    "tags": []
                },
                {
                    "url": "https:\/\/download.beckhoff.com\/download\/Document\/product-security\/Advisories\/advisory-2019-004.pdf",
                    "name": "https:\/\/download.beckhoff.com\/download\/Document\/product-security\/Advisories\/advisory-2019-004.pdf",
                    "refsource": "CONFIRM",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "When a Beckhoff TwinCAT Runtime receives a malformed UDP packet, the ADS Discovery Service shuts down. Note that the TwinCAT devices are still performing as normal."
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": [],
    "publishedDate": "2019-11-21T20:15Z",
    "lastModifiedDate": "2019-11-22T13:17Z"
}